.travis.yml

```diff
@@ -1,5 +1,7 @@
 language: ruby
 services: memcache
+before_install:
+  - gem update bundler
```

Comment:
https://travis-ci.org/Shopify/memcached_store/jobs/119200629

Fix `NoMethodError: undefined method `spec' for nil:NilClass`
This error is found in old bundler version.